default/python/README.md

```diff
@@ -28,10 +28,10 @@ We use [flake8-putty](https://pypi.python.org/pypi/flake8-putty)
 which include [flake8](https://pypi.python.org/pypi/flake8)
 which include [pycodestyle](https://pypi.python.org/pypi/pycodestyle) and other tools.
 
-`flake8-putty` is `flake8` plugin that allows one to disable rule for certain file,
-see `putty-ignore` section in `tox.ini`. This allow to ignore certain
-warnings only for specified files, like bare excepts, that hard to fie,
-or special formatting in table like files.
+`flake8-putty` is `flake8` plugin that allows one to disable one or more rules(s) for a certain file,
```

Comment:
I'm sorry, I had a typo in my previous suggestion.  This should be simply "one or more rules"

An alternate form would be "one or more rule(s)", but it is never proper to write "rules(s)"